src/components/dashboard/index.js

```diff
@@ -0,0 +1,329 @@
+/**
+ * @author johnworth
+ *
+ * The dashboard component.
+ *
+ * @module dashboard
+ */
+import React, { useState, useEffect } from "react";
+
+import { makeStyles } from "@material-ui/styles";
+import {
+    Button,
+    Card,
+    CardActions,
+    CardContent,
+    Divider,
+    Typography,
+} from "@material-ui/core";
+
+import { parse, format } from "date-fns";
+
+import { getMessage, withI18N, build as buildID } from "@cyverse-de/ui-lib";
+
+import messages from "./messages";
+import ids from "./ids";
+import * as constants from "./constants";
+
+const useStyles = makeStyles((theme) => ({
+    root: {
+        flexGrow: 1,
+    },
+    dividerRoot: {
+        marginBottom: 15,
+    },
+    footer: {
+        width: "100%",
+        height: 128, // This is needed to get the vertical scrolling to stop cutting off the bottom of the content.
+
+        [theme.breakpoints.down("sm")]: {
+            height: 32,
+        },
+    },
+    section: {
+        marginTop: 25,
+        marginBottom: 50,
```

Comment:
I will post just here but all `margin` and `spacing` should take value from `theme.spacing`.

Reply:
This should be fixed now.